prow/github/helpers.go

```diff
@@ -33,6 +33,16 @@ func HasLabel(label string, issueLabels []Label) bool {
 	return false
 }
 
+// HasLabels checks if all labels are in the label set "issueLabels"
+func HasLabels(labels []string, issueLabels []Label) bool {
```

Comment:
Only one package uses this function, please move this to that package and make it private to that package

Reply:
Done!